### ISSUE TITLE
Image Orientation Issue with keras-ocr on MacBook MPS

### ISSUE THREAD

**Issue by lasagna0** [faustomorales/keras-ocr#261]
## Description
When using keras-ocr with Metal Performance Shaders (MPS) on MacBooks (Apple Silicon), the OCR model appears to read images in reverse orientation, affecting text detection and recognition accuracy.

## Environment
- **Hardware**: MacBook with Apple Silicon (M1)
- **OS**: macOS
- **GPU Backend**: MPS (Metal Performance Shaders)
- **Library**: keras-ocr
- **TensorFlow Version**: 2.x
```

Im using the latest version of tensor flow-metal plugin v1.0.0 and TensorFlow v2.13, also the latest version of keras-ocr v0.8.4
import matplotlib.pyplot as plt
import keras_ocr
# keras-ocr will automatically download pretrained
# weights for the detector and recognizer.
pipeline = keras_ocr.pipeline.Pipeline()
# Get a set of three example images
images = [
keras_ocr.tools.read(url) for url in [
'https://upload.wikimedia.org/wikipedia/commons/b/bd/Army_Reserves_Recruitment_Banner_MOD_45156284.jpg','https://upload.wikimedia.org/wikipedia/commons/b/b4/EUBanana-500x112.jpg'
]
]
# Each list of predictions in prediction_groups is a list of
# (word, box) tuples.
prediction_groups = pipeline.recognize(images)
# Plot the predictions
fig, axs = plt.subplots(nrows=len(images), figsize=(20, 20))
for ax, image, predictions in zip(axs, images, prediction_groups):
    keras_ocr.tools.drawAnnotations(image=image, predictions=predictions, ax=ax)

```
![image](https://github.com/user-attachments/assets/0e5eeed0-11c4-4f87-8dcc-ffc3b5bb4101)

**Comment by lasagna0** [comment]
Error fixed using tensorflow-macos v2.10	and TensorFlow-metal v0.6.0 (downgrade)
